resources/stylesheet.windows.qss

```diff
@@ -0,0 +1,7 @@
+/********************************/
+/* Windows-specific style sheet for Webots */
+/********************************/
+
+QDialog * {
+  font-size: 13px;
```

Comment:
Now the `font-size` is set to the same value for all the platforms, so it is no longer platform specific.
Is it really this file the better place where to define it?

Reply:
We should probably define a common `stylesheet.qss` file to store common values, but this is probably out of the scope of this PR. I will open an issue for that.